source/_redirects

```diff
@@ -165,9 +165,11 @@
 /components /integrations
 
 # Renamed integrations
+/integrations/air_pollutants /integrations/air_quality
```

Comment:
Does this change really belong in the PR?

Reply:
Uh no, bit still broken on dev 😁

Reply:
Removed 👍